README.md

```diff
@@ -25,16 +30,16 @@ This can be achieved by starting Docker with the `-H` flag (e.g. `-H 0.0.0.0:237
 
 ```bash
 # create a cluster
-$ swarm create
```

Comment:
I wanted to stop people copying and pasting the example cluster ID, because that would be insecure and confusing, right?

My proposed approach does have the problem, however, that many of the commands you need to run get run in a different context (ie, on a different machine). So the environment variable isn't there.

Thinking about it, there are already some manually interpolated variables in this shell transcript. Let me do another commit to suggest a different way of spelling it.